<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WalletForm</class>
 <widget class="QDialog" name="WalletForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameMain">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <widget class="QToolButton" name="ButtonClose">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>21</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>✖️</string>
      </property>
     </widget>
     <widget class="QFrame" name="frameContent">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>361</width>
        <height>271</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
